app/controllers/general.php

```diff
@@ -278,6 +278,7 @@
         case 409: // Error allowed publicly
         case 412: // Error allowed publicly
         case 429: // Error allowed publicly
+        case 501: // Error allowed publicly
```

Comment:
If we're using the standard 400 status codes, should we do the same for 500 (501 is Not Implemented)? It doesn't make a functional difference, so long as we respond with an appropriate error description ( like [here](https://github.com/appwrite/appwrite/pull/947/files#diff-93771e9bb8e57cbe7d0c0e91e7aea5d85238a362f51caf957049f1d2ccc4d312R91))

Reply:
We allow 500 errors, but we hide the description with a generic message ("Server Error") to hide server errors in production.